mangopay/__init__.py

```diff
@@ -8,8 +8,8 @@
 mangopay_url_env_var = os.getenv("MANGOPAY_URL")
 
 if mangopay_url_env_var is not None:
-    api_url = f'http://{mangopay_url_env_var}/v2.01/'
-    api_sandbox_url = f'http://{mangopay_url_env_var}/v2.01/'
+    api_url = f'{mangopay_url_env_var}/v2.01/'
```

Comment:
this change was done as `MANGOPAY_URL` var always comes with a protocol - this was the root of the last week's issue (protocol was doubled)

Reply:
wait how did the old change  break production?